accounts/service.go

```diff
@@ -60,7 +64,7 @@ type InterceptorService struct {
 	invoiceToAccount map[lntypes.Hash]AccountID
 	pendingPayments  map[lntypes.Hash]*trackedPayment
 
-	mainErrChan chan<- error
+	mainErrChan chan<- func() (string, error)
```

Comment:
What if we instead just turn this into an `onError` callback? Then we don't need to do any filtering in Terminal and can just directly set the subserver status to errored.

Reply:
And we still don't need to add the non critical error queue.

Reply:
Updated with the latest push :)